assets/js/components/Work/Tabs/About/DescriptiveMetadata.jsx

```diff
@@ -19,29 +28,95 @@ const WorkTabsAboutDescriptiveMetadata = ({
   register,
   showDescriptiveMetadata,
 }) => {
-  const {
-    data: marcData,
-    loading: marcLoading,
-    errors: marcErrors,
-  } = useQuery(CODE_LIST_QUERY, { variables: { scheme: "MARC_RELATOR" } });
-  const {
-    data: subjectRoleData,
-    loading: subjectRoleLoading,
-    errors: subjectRoleErrors,
-  } = useQuery(CODE_LIST_QUERY, { variables: { scheme: "SUBJECT_ROLE" } });
-  const {
-    data: authorityData,
-    loading: authorityLoading,
-    errors: authorityErrors,
-  } = useQuery(CODE_LIST_QUERY, { variables: { scheme: "AUTHORITY" } });
+  const localList = localStorage.getItem("codeLists");
+  const [codeLists, setCodeLists] = useState(
+    localList ? JSON.parse(localList) : {}
+  );
+  useEffect(() => {
+    localStorage.setItem("codeLists", JSON.stringify(codeLists));
```

Comment:
This is going to set `localStorage.setItem` every single time the component loads.  We don't really want to re-set the `localStorage` value if it already exists, right?   Let's find a way around this.

Reply:
Maybe before calling line 36, we check 1.) `codeLists` doesn't exist or have values, or 2.) the user clicked the "refresh cache values" button?